index.js

```diff
@@ -241,26 +250,41 @@ function additionalProperty (ap) {
   return code
 }
 
-function addAdditionalProperties (ap) {
+function addAdditionalProperties (schema) {
   return `
       var keys = Object.keys(obj)
       for (var i = 0; i < keys.length; i++) {
         if (properties[keys[i]]) continue
-        ${additionalProperty(ap)}
+        ${additionalProperty(schema)}
       }
   `
 }
 
+function refFinder (ref, schema) {
+  // Split file from walk
+  ref = ref.split('#')
+  // If external file
+  if (ref[0]) {
+    schema = require(ref[0])
+  }
```

Comment:
Never do this, it's sync and done in a place where it might cause to bugs. We should not load files into Node. However, we should use an approach similar to https://www.npmjs.com/package/is-my-json-valid#external-schemas.